learn/avalanche-bridge-faq.md

```diff
@@ -120,6 +120,11 @@ You can use Metamask’s SWAP function to swap from ETH to WETH. Alternatively,
 
 The current-generation Avalanche Bridge \(AB\) to which this document refers is predated by a previous bridge implementation called the AEB. The AEB bridge and AB bridge each have their own unique token sets. The AEB tokens will be deprecated in the coming weeks in favor of the AB tokens. AB tokens have a `.e` suffix. While a token's name and symbol are good references to differentiate the two, the only surefire way to verify a token is the contract address. The AB token contract addresses can be found [here.](https://github.com/ava-labs/avalanche-bridge-resources/blob/main/avalanche_contract_address.json)
 
+#### Why doesn’t the newly bridged token appear on the portfolio automatically?
+
+It can't appear automatically because there are too many contracts running on the EVM. Metamask can't scan all of them to check if your address has a balance. Tokens are not physically in your C-chain address. They are "inside" the smart contract related to the token, and it's written in the chain that your address has xxx token. Your wallet (AVAX or Metamask) has to check all those contracts in order to see if there are some token belonging to you. But there are so many contracts it will take a lot of time, thus you have to manually tell to your wallet which contract it has to check.
+
```

Comment:
@jpop32 can you review this?

Reply:
Technically correct, but the wallet can check the common tokens, like the ones used by the bridge. It currently does so for the AEB tokens, I expect it makes sense to switch it over to AB tokens then. Emre can probably provide authoritative info.

Reply:
@kanatliemre can you help, please?

Reply:
Yes the web wallet (not metamask), has a default list of tokens supported out of the box. We can add the new bridge tokens there. Is there a tokenlist url for these tokens?

Reply:
Token list: https://github.com/ava-labs/avalanche-bridge-resources/blob/main/token_list.json